src/ViewHelpers/Format/SplitViewHelper.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file belongs to the package "TYPO3 Fluid".
+ * See LICENSE.txt that was shipped with this package.
+ */
+
+namespace TYPO3Fluid\Fluid\ViewHelpers\Format;
+
+use TYPO3Fluid\Fluid\Core\Rendering\RenderingContextInterface;
+use TYPO3Fluid\Fluid\Core\ViewHelper\AbstractViewHelper;
+use TYPO3Fluid\Fluid\Core\ViewHelper\Traits\CompileWithRenderStatic;
+
+final class SplitViewHelper extends AbstractViewHelper
+{
+    use CompileWithRenderStatic;
+
+    /**
+     * @var bool
+     */
+    protected $escapeChildren = false;
+
+    public function initializeArguments(): void
+    {
+        $this->registerArgument('delimiter', 'mixed', 'Delimiter string to explode with', true);
+        $this->registerArgument('value', 'string', 'The string to explode');
+        $this->registerArgument('limit', 'int', 'If limit is positive, a maximum of $limit items will be returned. If limit is negative, all items except for the last $limit items will be returned. 0 will be treated as 1.', false, PHP_INT_MAX);
+        $this->registerArgument('trim', 'bool', 'Trim items in array', false, false);
```

Comment:
No need to overcomplicate stuff, there is no need for a splitter to trim stuff.

Reply:
My $0.02 on this: Use cases for integrators might be to receive a string like:

"cars, dogs, cats"

and they might want to put that into `data` attributes for an `<a>` tag or something like that. So they would need to trim each variable.

When the split viewhelper would not have a trim, we'd have these options:

* Do not use a viewhelper, but prepare the data in PHP scope (controller, dataprocessor, ...) -> might need more developer skills than integrator

* Create a <f:trim> viewhelper -> I think the more "small" viewhelpers we have then, that need chaining, the more fluid would be bent to contain more much logic. You'd have the whole split, foreach, trim logic there.

So this parameter to directly trim values (like in PHP the trimExplode does for us) does have some value for integrators to save one "logic" operation.

Reply:
In Twig, you'd do something like `{{ array|map(v => v|trim) }}`. Not sure if this would be possible in Fluid at all.

Reply:
> In Twig, you'd do something like `{{ array|map(v => v|trim) }}`. Not sure if this would be possible in Fluid at all.

I don't think a clean solution like this is possible without syntax changes. Something like this could be possible though:

```xml
{csv -> f:split(separator: ',') -> f:map(callback: 'f:trim')}
```